.env.template

```diff
@@ -3,4 +3,4 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 
 # Change the following values for local development.
-SECRET_KEY=secret-key
+SECRET_KEY=CBC93B89-EADA-4B2B-A90F-8982D888F047
```

Comment:
I hope this isn't the one you're actually planning to use!

Reply:
Whoops meant to change it in `.env`!